src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => (
```

Comment:
Single line is okay here.
```suggestion
export const Article = ({ title, date, text }) => (
```